libs/components/router/src/lib/modules/href/href.directive.spec.ts

```diff
@@ -76,6 +76,7 @@ describe('SkyHref Directive', () => {
     expect(links.filter((e: HTMLElement) => !!e.offsetParent).length).toEqual(
       6
     );
+    flush();
```

Comment:
Instead of doing these `flush` calls at the end of all these tests is there harm in just adding them to an `afterEach` so that we don't have to remember that line on any new test?